core/frontend/src/RealityDataSourceCesiumIonAssetImpl.ts

```diff
@@ -84,24 +84,14 @@ export class RealityDataSourceCesiumIonAssetImpl implements RealityDataSource {
     else
       this._baseUrl = `${urlParts.join("/")}/`;
   }
-  private async _doRequest(url: string, responseType: string): Promise<any> {
-    let options: RequestOptions = {
-      method: "GET",
-      responseType,
+  private async _doRequest(url: string, responseType: "json" | "arraybuffer"): Promise<any> {
```

Comment:
do we need to keep this?

Reply:
Removed it. Without this, code will be a bit more type safe.